controllers/chat/group/changeChannelDetail.js

```diff
@@ -44,6 +44,9 @@ const changeChannelDetail = async (req, res) => {
         other_text: textTargetUser,
         type: 'system',
         user_id: userId,
+        only_to_user_show: userId,
+        disable_to_user: false,
+        is_from_prepopulated: true,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new properties `only_to_user_show`, `disable_to_user`, and `is_from_prepopulated` have been added. However, it's not clear what these properties are intended for as there is no context or comments explaining their purpose. If they are meant to control visibility or access, consider renaming them to be more descriptive and intuitive. Also, ensure that these properties are being used correctly throughout the codebase.

```diff
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
+        showOnlyToUser: userId,
+        disableForUser: false,
+        isPrepopulated: true,
```

Please note that this suggestion assumes that the new names accurately reflect the intended functionality of these properties. You should adjust the names according to your specific use case.


<!-- This is an auto-generated comment by OSS CodeRabbit -->